packages/driver/src/conUtils.ts

```diff
@@ -663,6 +688,10 @@ async function parseConnectDsnAndArgs(
           .readFileUtf8(stashDir, "database")
           .then((name) => name.trim())
           .catch(() => undefined),
+        serverUtils
+          .readFileUtf8(stashDir, "branch")
+          .then((name) => name.trim())
+          .catch(() => undefined),
```

Comment:
We're not assigning this to a variable on line 682. Are we missing a test case for this in the shared tests?

Reply:
oh yes, u are right